src/cmem/ActivityControl/ActivityControlWidget.tsx

```diff
@@ -75,78 +100,91 @@ export function ActivityControlWidget(props: IActivityControlProps) {
         activityContextMenu,
         small,
         border,
-        canShrink
-    } = props
-
-    const spinnerClassNames = (progressSpinner?.className ?? "") + ` ${eccgui}-spinner--permanent`
+        canShrink,
+        tags,
+    } = props;
 
+    const spinnerClassNames = (progressSpinner?.className ?? "") + ` ${eccgui}-spinner--permanent`;
     const widget = (
         <OverviewItem data-test-id={dataTestId} hasSpacing={border} densityHigh={small}>
-            {progressBar && (
-                <ProgressBar
-                    {...progressBar}
-                />
-            )}
+            {progressBar && <ProgressBar {...progressBar} />}
             {progressSpinner && (
                 <OverviewItemDepiction keepColors>
-                    <Spinner
-                        position="inline"
-                        size={small ? "tiny" : "small"}
-                        stroke={small ? "bold" : "medium"}
-                        {...progressSpinner}
-                        className={spinnerClassNames}
-                    />
+                    {progressSpinner?.value ? (
```

Comment:
This is wrong. It prevents to show progress 1 to 100 via Spinner element because it is replaced by an static icon.

Reply:
It woukd probably better to extend the `IActivityControlProps` interface for an icon/depiction property and if this is set then the spinner is replaced, in case there is one.